README.md

```diff
@@ -9,60 +9,42 @@ with Bluetooth Low Energy devices.
-
-```kotlin
-val central = scope.central(context)
-```
+## Scanning
 
-### JavaScript & Apple
+To scan for nearby peripherals, the `Scanner` provides an `advertisements` `Flow` which is a stream of `Advertisement`
+objects representing advertisements seen from nearby peripherals. `Advertisement` objects contain information such as
+the peripheral's name and RSSI (signal strength).
 
-For both JavaScript and Apple-based targets, the `central` extension function is called without arguments:
+Scanning begins when the `advertisements` `Flow` is collected and stops when the `Flow` collection is terminated. A
+`Flow` terminal operator (such as [`first`]) may be used to scan until an advertisement is found that matches a desired
+predicate. 
 
 ```kotlin
-val central = scope.central()
+val advertisement = Scanner()
+    .advertisements
+    .first { it.name?.startsWith("Example") }
 ```
 
-## Scanning
+_**JavaScript:** Scanning for nearby peripherals is supported, but only available on Chrome 79+ with "Experimental Web
+Platform features" enabled via:_ `chrome://flags/#enable-experimental-web-platform-features`
 
-To scan for nearby peripherals, acquire a `Scanner` from the `Central` via the `scanner` function. To begin scanning,
-simply collect from the `Scanner`'s `peripheral` `Flow`. The `peripheral` `Flow` provides a stream of `Advertisement`
-objects representing advertisements seen from nearby peripherals. When `Flow` collection is terminated, scanning will
-stop. A `Flow` terminal operator (such as [`first`]) may be used to scan until an advertisement is found which matches a
-desired predicated.
-
-Using an `Advertisement` object, you can acquire a `Peripheral` using the `Central.peripheral` function.
+## Peripheral
 
-### Android & Apple
+Once an `Advertisement` is obtained, it can be converted to a `Peripheral` via the `CoroutineScope.peripheral` extension
+function. `Peripheral` objects represent actions that can be performed against a remote peripheral, such as connection
+handling and I/O operations.
 
 ```kotlin
-val scanner = central.scanner()
-val advertisement = scanner.peripherals.first { it.name?.startsWith("Example") }
-val peripheral = central.peripheral(advertisement)
+val peripheral = scope.peripheral(advertisement)
 ```
 
 ### JavaScript
 
-On JavaScript, the standard paradigm is to request a specific peripheral rather than processing a stream of
-advertisements. Criterium (`Options`) such as expected service UUIDs on the peripheral and/or the peripheral's name may
-be specified. When `requestPeripheral` is called with the specified options, the browser shows the user a list of
-peripherals matching the criterium. The peripheral chosen by the user is then returned (as a `Peripheral` object).
+On JavaScript, rather than processing a stream of advertisements, a specific peripheral can be requested using the
+`CoroutineScope.requestPeripheral` extension function. Criterium (`Options`) such as expected service UUIDs on the
```

Comment:
super nit: singular of criteria is criterion, though I think criteria is correct here. (Had to google it myself)

Reply:
Good catch. I was intending it to be plural (as it's representing multiple options), so "criteria" is what should be here.
Not sure where I got criterium from, looks to be a [bicycle race](https://en.wikipedia.org/wiki/Criterium). 🤷